README.md

```diff
@@ -86,6 +88,31 @@ react-native link react-native-pdf
 
 </details>
 
+### Windows installation
+<details>
+  <sumary>Windows details</summary>
+
+- Open your solution in Visual Studio 2019 (eg. `windows\yourapp.sln`)
+- Right-click Solution icon in Solution Explorer > Add > Existing Project...
+- Add `node_modules\@react-native-community\progress-view\windows\progress-view\progress-view.vcxproj`
+- If running RNW 0.62: add `node_modules\react-native-pdf\windows\RCTPdf\RCTPdf.vcxproj`
```

Comment:
@bzoz @jaimecbernardo - Why is this a 0.62 note? It looks like this is required even in 0.63?

Reply:
This action isn't needed for RNW 0.63 for RCTPdf, since auto-linking does it.
Adding the `#include "winrt/RCTPdf.h"` line is still needed, though, due to the existing XAML component or there'll be an error on `XamlTypeInfo.g.cpp`.

Manual linking is still needed for the `@react-native-community/progress-view` module though.